api/app/services/demonstration/data_loader.rb

```diff
@@ -70,6 +77,25 @@ def create_pages
       end
     end
 
+    def ensure_settings
+      settings = Settings.instance
+      ENV_SETTINGS.each do |key, values|
+        values.each do |value|
+          env_setting = "MANIFOLD_SETTING_#{key}_#{value}".upcase
+          settings[key][value] = settings[key][value] ||= ENV[env_setting]
+          @logger.info("Setting updated: #{key}: #{value}".green)
```

Comment:
This log output occurs whether or not the setting was updated, and it doesn't include any information about what it was set to. Not very useful.